components/ntp_sponsored_images/browser/locale_helper.cc

```diff
@@ -0,0 +1,22 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/ntp_sponsored_images/browser/locale_helper.h"
+
+namespace internal {
```

Comment:
please add a dep for brave_ads and use the existing locale helper. We should not have two different singletons for this

Reply:
done